terasoluna-gfw-parent/pom.xml

```diff
@@ -414,6 +434,10 @@
     <cargo.container.id>tomcat8x</cargo.container.id>
     <cargo.container.url>http://archive.apache.org/dist/tomcat/tomcat-8/v${cargo.tomcat8.version}/bin/apache-tomcat-${cargo.tomcat8.version}.zip</cargo.container.url>
     <cargo.tomcat8.version>8.5.20</cargo.tomcat8.version>
+    <!-- == Spring Framework== -->
+    <springframework.version>4.3.14.RELEASE</springframework.version>
+    <!-- == Spring Security == -->
+    <springframework.security.version>4.2.4.RELEASE</springframework.security.version>
```

Comment:
- [ ] Similar to `dependencyManagement`.

Reply:
OK